src/agent0/ethpy/hyperdrive/transactions.py

```diff
@@ -118,61 +120,65 @@ def get_hyperdrive_checkpoint_exposure(
 
 @overload
 def parse_logs_to_event(
-    tx_receipt: TxReceipt, hyperdrive_contract: Contract, fn_name: Literal["createCheckpoint"]
+    tx_receipt: TxReceipt, hyperdrive_interface: HyperdriveReadInterface, fn_name: Literal["createCheckpoint"]
```

Comment:
I think `interface` instead of `hyperdrive_interface` is what we use everywhere else in ethpy